common/result.c

```diff
@@ -81,8 +81,8 @@ const char* oe_result_str(oe_result_t result)
             return "OE_MEMORY_LEAK";
         case OE_BAD_ALIGNMENT:
             return "OE_BAD_ALIGNMENT";
-        case OE_TCB_INFO_PARSE_ERROR:
-            return "OE_TCB_INFO_PARSE_ERROR";
+        case OE_JSON_INFO_PARSE_ERROR:
```

Comment:
Since we are reusing the paring routine in the TCBinfo file, which makes them generic.
If we want to using a new error code, we would need to pass in default error code depending on where those APIs were called from, which  would make the code very wordy.

Reply:
Ok, I agree.
It would be good to update the associated comment for OE_JSON_INFO_PARSE_ERROR in bits/result.h.